paddle/fluid/operators/mkldnn/activation_mkldnn_op.cc

```diff
@@ -73,8 +73,12 @@ void eltwise_forward(const framework::ExecutionContext &ctx,
   const auto *x = ctx.Input<Tensor>("X");
   auto *y = ctx.Output<Tensor>("Out");
 
-  const T alpha = ctx.HasAttr("alpha") ? ctx.Attr<T>("alpha") : 0;
-  const T beta = ctx.HasAttr("beta") ? ctx.Attr<T>("beta") : 0;
+  T alpha = ctx.HasAttr("alpha") ? ctx.Attr<T>("alpha") : 0;
+  T beta = ctx.HasAttr("beta") ? ctx.Attr<T>("beta") : 0;
+
+  if (algorithm == mkldnn::algorithm::eltwise_swish) {
```

Comment:
It may be helpful to add a comment here explaining why `alpha` and `beta` need to be swapped.

Reply:
ok I will